config/initializers/country_config.rb

```diff
@@ -44,6 +44,7 @@ class CountryConfig
         cumulative_registrations: "bdMBWIf2i1h",
         monthly_registrations: "tI9g0mtCzOv"
       },
+      enabled_diabetes_population_coverage: true,
```

Comment:
This is not a feature flag. Diabetes patient coverage is supposed to be turned off for India, but remain on for BD/ET/SL etc. Since this is country specific, we've added it to the country config.